extensions/interactions/TextInput/TextInputValidationService.js

```diff
@@ -17,12 +17,36 @@
  */
 
 oppia.factory('TextInputValidationService', [
-  'baseInteractionValidationService',
-  function(baseInteractionValidationService) {
+  'WARNING_TYPES', 'baseInteractionValidationService',
+  function(WARNING_TYPES, baseInteractionValidationService) {
     return {
       getCustomizationArgsWarnings: function(customizationArgs) {
-        // TODO(juansaba): Implement customization args validations.
-        return [];
+        var warningsList = [];
+        baseInteractionValidationService.requireCustomizationArguments(
+          customizationArgs,
+          ['placeholder', 'rows']);
+
+        var rows = customizationArgs.rows.value;
+        var placeholder = customizationArgs.placeholder.value;
+        if (typeof placeholder != 'string') {
+          warningsList.push({
+            type: WARNING_TYPES.ERROR,
+            message: (
+              'Placeholder text must be a string.')
+          });
+        }
+        // Can we get these values dynamically?
```

Comment:
That's a great question. I think we have a constant called INTERACTION_SPECS -- could you check whether the necessary customization arg info is in there?